[API] intro fix to drop mention of customer creation and brand count. (#112)
UnumID/UnumID.github.io: docs/acceptance-guide.mdx

  </details>

  <details>
    <summary>Brand Accepting Credentials via Partner Acceptance</summary>
    <p>

>       This demo shows Lendtable accepting their partner, Hooli, credentials in order to gain 1-Click IDV capabilities. In this case Hooli can issue the credential in an on-demand, aka "partner", fashion when the user clicks the button to redeem the Lendtable offer in the Hooli dashboard. This would ensure the user has the necessary credentials, so Lendtable can verify with 1-Click.

Reviewer: ```suggestion
      This demo shows Lendtable accepting credentials from their partner, Hooli, in order to gain 1-Click IDV capabilities. Because Hooli has partnered with Lendtable, they can issue the necessary credentials on-demand when the user clicks the button to redeem the Lendtable offer in the Hooli app, ensuring Lendtable can verify with 1-Click.
```
